init.lua

```diff
@@ -135,23 +133,54 @@ local function item_has_groups(item_groups, groups)
 	return true
 end
 
-local function groups_to_item(groups)
-	if #groups == 1 then
-		local group = groups[1]
-		if group_stereotypes[group] then
-			return group_stereotypes[group]
-		elseif reg_items["default:" .. group] then
-			return "default:" .. group
+local function get_group_items(str)
+	local group_string = str:sub(7)
```

Comment:
Use the name `group`

Reply:
`group` is used bellow for resolved group. I need 2x variable names for `group:flower,color_red` and `flower`. Maybe other variable name?